Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-buster as build-stage
+FROM golang:1.18-buster as build-stage
```

Comment:
Weird since 1.18-buster is the same as 1.18.6-buster. So it should not complain about the vulnerability

Reply:
Hmm, let me try pinning to specifically 1.18.6, maybe the tool doesn't know what 1.18 version is actually pulled in(not sure if the scan is just statically analyzing the FROM statement.